descriptors/ModuleDescriptor-template.json

```diff
@@ -677,12 +677,15 @@
       "permissions": [
         "inventory-storage.instances.item.get",
         "inventory-storage.instances.item.put",
-        "mapping-metadata.get",
+        "mapping-metadata.item.get",
+        "mapping-metadata.type.item.get",
         "mapping-rules.get",
-        "source-storage.sourceRecords.get",
+        "source-storage.stream.source-records.collection.get",
+        "source-storage.source-records.item.get",
+        "source-storage.source-records.collection.get",
```

Comment:
The module uses only `source-storage/source-records/{id}` and `/batch/parsed-records/fetch` endpoints, so "source-storage.source-records.collection.get" and "source-storage.stream.source-records.collection.get" permissions could be removed.